jalib/jalib.cpp

```diff
@@ -34,29 +34,56 @@
 #include "jalib.h"
 #include "jassert.h"
 
-jalib::JalibFuncPtrs jalib::jalibFuncPtrs;
-int jalib::jalib_funcptrs_initialized = 0;
-const char *jalib::elfInterpreter = NULL;
-int jalib::stderrFd = -1;
-int jalib::logFd = -1;
-int jalib::dmtcp_fail_rc = -1;
+static jalib::JalibFuncPtrs jalibFuncPtrs;
+static int jalib_funcptrs_initialized = 0;
+
+static struct {
+  const char *elfInterpreter;
+  int stderrFd;
+  int logFd;
+  int dmtcp_fail_rc;
+} dmtcpInfo = {NULL, -1, -1, -1};
```

Comment:
I would suggest that we do it for the entire code base. I will try to come up with a pull request soon.